docs/content/patterns/helpful-form-content/index.mdx

```diff
@@ -0,0 +1,262 @@
+				isOpen={isDrawerOpen}
+				onClose={closeDrawer}
+				title="Inviting a staff member"
+				width="lg"
+			>
+				<Stack gap={1.5}>
+					<Text as="p" color="muted" fontSize="md">
+						Inviting individuals to join your staff can bring numerous benefits
+						to your business and help streamline your operations.
+					</Text>
+					<Text as="p">
+						As your business grows, you’ll need to scale your team accordingly.
+						Inviting new staff members allows you to expand your workforce in
+						line with your evolving needs.
+					</Text>
+					<Text as="p">Below are a list of the types of roles available:</Text>
+					<H2>Owner</H2>
+					<Text as="p">
+						Owners have the highest level of access. They can add, edit and
+						remove managers and staff members. They can set rosters, send an
+						receive messages with owners, employees and other managers.
+					</Text>
+					<H2>Manager</H2>
+					<Text as="p">
+						Managers can add, edit and remove staff members. They can set
+						rosters, send an receive messages with owners, employees and other
+						managers.
+					</Text>
+					<H2>Employee</H2>
+					<Text as="p">
+						Employees can see their shifts, send an receive messages with
+						managers and edit their personal and contact information.
+					</Text>
+					<Callout title="Need help?">
+						<Text as="p">
+							Call <strong>02 9481 1122</strong> (9am to 5pm AEST Monday to
+							Friday)
+							<br />
+							Email{' '}
+							<TextLink href="mailto:emailaccount@yourgov.gov.au">
+								emailaccount@yourgov.gov.au
+							</TextLink>
+						</Text>
+					</Callout>
+				</Stack>
+			</Drawer>
+		</>
+	);
+};
+```
+
+## Drawer
+
+A Drawer is a panel that slides in from the right side of the screen. The Drawer is overlayed on top of the main area of the page to capture the user’s attention while keeping the context of the current task.
+
+Drawers can be used to filter data, for small subtask forms and to provide supporting information to users when completing a task.
+
+Avoid lengthy content that would result in scrolling.
+
+Explore the [Drawer](/components/drawer) options to see if they might be suitable.
+
+```jsx live
+() => {
+	/** import { VisuallyHidden } from '@ag.ds-next/react/a11y'; */
+	const [isDrawerOpen, openDrawer, closeDrawer] = useTernaryState(false);
+	return (
+		<>
+			<Text as="p">
+				Learn more about{' '}
+				<TextLink
+					href="/help/access-requests"
+					onClick={(event) => {
+						if (event.metaKey) return;
+						event.preventDefault();
+						openDrawer();
+					}}
+				>
+					understanding access requests
+					<VisuallyHidden>, Opens in a side drawer</VisuallyHidden>
+					<Box
+						as="span"
+						css={{
+							display: 'inline',
+							marginLeft: '0.25rem',
+							position: 'relative',
+							top: 3,
+						}}
+					>
+						<MoveLeftIcon size="sm" weight="regular" />
+					</Box>
+				</TextLink>{' '}
+				for your staff
```

Comment:
I don't think we should put anything after the link, otherwise the sentence is "Learn more about understanding access requests, Opens in a side drawer for your staff
```suggestion
```